inst/app/server.R

```diff
@@ -10,4 +10,31 @@
 
 server <- function(input, output, session) {
 
+  # ----- REACTIVES ----- #
+
+
+  # ----- OBSERVE EVENTS ----- #
+
+
+  # ----- TAB: Collision Location Map ----- #
+
+
+  # RenderPlot: Districts ---------------------------------------------------
+  source(file = "modules/plotly_testvisual.R", local = TRUE)
```

Comment:
Fantastic to see you using modules from the very start - it's really useful in terms of chunking up and isolating independent pieces of logic.

Reply:
Thanks for your tips indeed!